dev-infrastructure/templates/mgmt-cluster.bicep

```diff
@@ -107,6 +107,11 @@ module mgmtCluster '../modules/aks-cluster-base.bicep' = {
         namespace: 'maestro'
         serviceAccountName: 'maestro'
       }
+      package_operator: {
```

Comment:
do we actually need a MI for PKO? afaik it does not interact with Azure?

Reply:
I configured it, cause I saw issues with pulling images. But I can try another way, perhaps I can get rid of it

Reply:
note required